src/fsearch_window_actions.c

```diff
@@ -384,6 +398,7 @@ fsearch_window_action_open (GSimpleAction *action,
             gtk_tree_selection_selected_foreach (selection, open_cb, NULL);
         }
     }
+    fsearch_window_action_after_file_open(false);
```

Comment:
This action should only be called when the file actually was opened. Right now the application even gets quit when no attempt to open a  file occurred, when more than 10 files were selected (yes, this arbitrary limit sucks, I'll need to add a configurable confirm dialog which at least warns the user when he tries to open lots of files). So I guess we should add some error handling here and only trigger the quit/minimize action when at least one file was successfully opened.